src/stories/30_Visual_Effects/Ken_Burns.html

```diff
@@ -0,0 +1,143 @@
+* Animate the `<amg-img>` element using the `animate-in="pan-right"`. (You can also use `pan-left`, `pan-up`, and `pan-down`. [See amp-story animations list.](/stories/features/animations/#zoom-in) 
+
+  
+<div class="ampstart-card info">
+**Tip:** It's important that the **image** sets the **panning animation**, whereas the image **container** sets the **zoom animation**. Otherwise the dimensions of the image will change, causing undesired results in the animation.
+</div>
+    -->
+    <amp-story-page id="ken-burns-effect1">
+      <amp-story-grid-layer template="vertical">
+        <div animate-in="zoom-in" animate-in-duration="30s" class="img-container">
+            <amp-img id="ken-burns-img1"
+              src="https://picsum.photos/1600/1200?image=1077"
+              animate-in="pan-left"
+              animate-in-duration="30s"
+              layout="fixed"
+              width="1600"
+              height="1200">
+            </amp-img>
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Using Multiple Images -->
+    <!-- 
+      You can also achieve the effect with multiple images!
+      
+      Just combine the animations as described in the [combining animations section](/stories/features/animations/#combining-animations). Remember all the elements being animated after certain animation must include the `animate-in-after` attribute. Including the children being animated. See below for the code sample.
+          
+      Keep in mind results may vary depending on the image size and animations you use. Experiment with different animation durations and presets to see what you like best!
+    -->
+    <amp-story-page id="ken-burns-effect2">
+        <amp-story-grid-layer template="vertical">
+          <div animate-in="zoom-in" animate-in-duration="30s" class="img-container">
+              <amp-img id="ken-burns-img1"
+                src="https://picsum.photos/1600/1200?image=1078"
+                animate-in="pan-right"
+                animate-in-duration="30s"
+                layout="fixed"
+                width="1600"
+                height="1200">
+              </amp-img>
+          </div>
+          <div animate-in="fade-in" animate-in-after="ken-burns-img1" animate-in-duration="1s">
+            <div
+              animate-in="zoom-out"
+              animate-in-duration="30s"
+              class="img-container"
+              animate-in-after="ken-burns-img1">
+              <amp-img id="ken-burns-img2"
+                src="https://picsum.photos/1600/1200?image=1026"
+                layout="fixed"
+                width="1600"
+                height="1200"
+                animate-in="pan-down"
+                animate-in-duration="30s"
+                animate-in-after="ken-burns-img1">
+              </amp-img>
+            </div>
+          </div>
+        </amp-story-grid-layer>
```

Comment:
It'd be nice to show 1 or 2 more images - what do you think?

Reply:
Sounds good!